bim2sim/sim_settings.py

```diff
@@ -399,7 +512,15 @@ def __init__(self):
                     " the walls of the selected building.",
         for_frontend=True
     )
-    construction_class_windows = Setting(
+    year_of_construction_overwrite = NumberSetting(
+        default=None,
+        min_value=1918,
```

Comment:
min value should be 0 (this is the minimum value allowed in the templates).

```suggestion
        min_value=0,
```

Reply:
solved